mkosi.images/initrd/mkosi.conf.d/debian-ubuntu.conf

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=|debian
+Distribution=|ubuntu
+
+[Content]
+Packages=
+        btrfs-progs
+        tpm2-tools
+
+VolatilePackages=
+        systemd
```

Comment:
Pulled in as a dep so it doesn't really matter for the initrd. It only matters to list them for the main image because we also read all the volatile packages in the prepare script there to retrieve recommended and suggested deps

Reply:
I'll do another PR to add them everywhere just to be sure